commands/cdn/distribution/create.go

```diff
@@ -0,0 +1,113 @@
+package distribution
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/pointer"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/uuidgen"
+
+	cdn "github.com/ionos-cloud/sdk-go-cdn"
+
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/spf13/viper"
+
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+)
+
+func Create() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
```

Comment:
I believe it might actually be more appropriate to have the `create` command use a json properties file to create the object. From what I can tell, the routing rules have quite a complex structure and it might be easier for the user to just use a properties file.